docs/src/pages/about.md

```diff
@@ -55,8 +23,28 @@ People from the following projects and organizations are collaborating on this p
 | [SpotBugs](http://spotbugs.rtfd.io/)                    | [SpotBugs Team](https://github.com/spotbugs/) |
 | (various)                                               | [Square](https://squareup.com)                |
 
+## What are we trying to do?
+
+Java static analysis tools, such as Error Prone and those built into IDEs, have been very successful
+at enabling automated refactorings and bug prevention.
+
+Many of their most beneficial analyses require certain Java annotations (such as `@Nullable` or
+`@Immutable`) to be present in the source or class file. Yet for the going-on-two-decades since 
```

Comment:
nit: "source or class file being analyzed" or pluralize it: "in source or class files"

Reply:
done